### PR TITLE
QU-1825 styles, utility classes and grid changes needed for CXM integration mapping

### PR DESCRIPTION
**Pull request by jamesjacobs**

### CODE REVIEW

File: stylesheets/_mixin.responsive-utilities.scss [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+@mixin sr-visible-on-phone {
+    @include respond-min($screen-phone) {
+        @include sr-hide;
+    }
+}
+
+@mixin visible-on-tablet {
+    @media not all and (min-width: $screen-tablet) and (max-width: $screen-desktop - 1) {
```
Comment (langy) [inline stylesheets/_mixin.responsive-utilities.scss]: Use `@include respond-min-max($screen-tablet, $screen-desktop)` here instead?

File: stylesheets/_mixin.responsive-utilities.scss [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        @include hide;
+    }
+}
+
+@mixin sr-visible-on-tablet {
+    @media not all and (min-width: $screen-tablet) and (max-width: $screen-desktop - 1) {
```
Comment (langy) [inline stylesheets/_mixin.responsive-utilities.scss]: Use `@include respond-min-max($screen-tablet, $screen-desktop)` here instead?

File: stylesheets/_override.shame.scss [hunk 3]
```diff
@@ -202,3 +202,10 @@
 .lt-ie10 .content-main {
     clear: both;
 }
+
+.icon--align-with-fields {
+    display: block;
+    font-size: 24px;
+    line-height: 48px;
```
Comment (langy) [inline stylesheets/_override.shame.scss]: Could the above two lines use vars instead?

File: stylesheets/_mixin.responsive-utilities.scss (hunk 2 above)
Comment (langy) [inline stylesheets/_mixin.responsive-utilities.scss]: Ignore these, just realised it's a hack to get it to do the reverse logic. Could we write a new mixin for this, or amend the `respond-min-max` one to include a check that adds this for you?

File: stylesheets/_override.shame.scss [hunk 4]
```diff
@@ -202,3 +202,10 @@
 .lt-ie10 .content-main {
     clear: both;
 }
+
+.icon--align-with-fields {
```
Comment (Stanton) [inline stylesheets/_override.shame.scss]: Is this going to be/should this be resolved in a non-override way? Need a new ticket?

File: views/playground/responsive-utility-classes/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,70 @@
+{% extends '@pulsar/pulsar/layouts/base.html.twig' %}
```
Comment (Stanton) [inline views/playground/responsive-utility-classes/index.html.twig]: Would this playground be more appropriate as a new section in the Lexicon and GitBook?

File: stylesheets/_mixin.media-queries.scss [hunk 6]
```diff
@@ -22,15 +22,22 @@
...
     @if $ie-version < 9 {
         @if $max-width >= $screen-medium {
             @content;
         }
     }
     @else {
-        @media screen and (min-width: $min-width) and (max-width: $max-width - 1) {
-            @content;
+        @if ($reverse) {
+            @media not screen and (min-width: $screen-tablet) and (max-width: $screen-desktop - 1) {
```
Comment (langy) [inline stylesheets/_mixin.media-queries.scss]: Needs vars updating to mixin ones

File: stylesheets/_override.shame.scss (hunk 4 above)
Comment (jamesjacobs) [inline stylesheets/_override.shame.scss]: As discussed, could be replaced with font size and line height utility classes but feels like bloat for a rare situation. Leaving as is.

File: views/playground/responsive-utility-classes/index.html.twig (hunk 5 above)
Comment (jamesjacobs) [inline views/playground/responsive-utility-classes/index.html.twig]: Will add a new ticket for a utilities section in the lexicon/gitbook. Will need some refactoring of blocks due to non inheritance of the javascript block in tabs

**Pull request merged**